src/components/notification/notification-list.component.ts

```diff
@@ -99,8 +99,38 @@ export class NotificationListComponent implements AfterViewInit, OnDestroy {
             const element = elements[i].nativeElement;
             const notification = notifications[i];
             this._renderer.setStyle(element, 'top', `${top}px`);
-            top = top + element.scrollHeight + notification.spacing;
+            top = top + this.getNotificationHeightPixels(notification, elements[i]);
+        }
+    }
+
+    private updateListPosition(elements: ElementRef[], notifications: NotificationRef[]): void {
+        if (this.position === 'bottom-left' || this.position === 'bottom-right') {
+
+            // calculate the total height of all notifications including spacing
+            this._bottom = notifications.reduce((total, notification, index) =>
```

Comment:
I think `totalHeight` would be a better name here. Then you could also remove the comment as it would then be clearer what the value represents.

Reply:
If you mean the `_bottom` property, that's used to derive the CSS `bottom` value (see above) so renaming it would be contradictory. The comment probably causes the confusion here so I have removed it now.

Reply:
```suggestion
            this._bottom = notifications.reduce((totalHeight, notification, index) =>
```
No, I meant the `total` within `reduce` could possibly be changed to `totalHeight` to give a future reader of this line a quicker understanding of its purpose

Reply:
Oh, yes, that's reasonable. Thanks.